integration/src/test/scala/ammonite/integration/LineNumberTests.scala

```diff
@@ -13,36 +14,48 @@ object LineNumberTests extends TestSuite{
   val tests = this{
 
     def checkErrorMessage(file: RelPath, expected: String) = {
-      val e = intercept[ShelloutException]{ 
+      val e = intercept[ShelloutException]{
         exec(file)
-      }.result.err.string.replace("\r", "").replace("\n", System.lineSeparator())
-      assert(e.contains(expected.replace("\n", System.lineSeparator())))
+      }.result.err.string
+      assert(e.contains(expected.replace("\n", Util.newLine)))
```

Comment:
I have changed for this instance but there are numerous other instances in test expected outputs where `.replace("\n", Util.newLine` is used as strings written on linux appear different as I mentioned above. I think replace will be ok and natural for those places like this
`
          s"""The following arguments failed to be parsed:
             |(i: Int) failed to parse input "foo" with $exMsg
             |expected arguments: (i: Int, s: String, path: ammonite.ops.Path)"""
            .stripMargin
            .replace("\n", Util.newLine)
`
If you say I will change them also